tcms/xmlrpc/api/testplan.py

```diff
@@ -735,7 +722,7 @@ def import_case_via_XML(request, plan_id, values):
 
     try:
         new_case_from_xml = clean_xml_file(values)
-    except:
+    except ValueError:
```

Comment:
`except Exception` in this case. While `clean_xml_file` only raises ValueError it calls bytes.decode() and also does XML parsing both of which may raise exception we're unaware of.

Reply:
Done